src/localization/fr/general.json

```diff
@@ -0,0 +1,35 @@
+{
+  "general.prev": "Précédent",
+  "general.next": "Suivant",
+  "general.result": "Résultat",
+  "general.regex": "Regex",
+  "general.text": "Texte",
+  "general.start": "Démarrer",
+  "general.continue": "Continuer",
+  "general.completedStep": "Etape terminé!",
+  "general.hintQuestion": "Montrer la réponse",
+  "general.reportStep": "Signaler un problème",
+  "general.startLearning": "Commencer à apprendre",
+  "general.comingSoon": "A venir",
+  "general.becomeSponsor": "Devenir un sponsor",
+  "general.learn": "Apprendre",
+  "general.playground": "Terrain de jeux",
+  "general.cheatsheet": "Aide-mémoire",
+  "general.ourSponsors": "Nos sponsors",
+
+  "notFound.intro": "La page que vous recherchez n'existe pas.",
+  "notFound.button": "Retour à l'accueil",
+
+  "page.landing.title": "Regex Learn - Pas à pas, de zéro à avancé.",
+  "page.learn.title": "Regex Learn - Cours interactif sur Regex",
+  "page.cheatsheet.title": "Regex Learn - Regex Aide-mémoire",
+  "page.playground.title": "Regex Learn - Terrain de jeux",
+  "page.404.title": "Regex Learn - Non trouvé",
+
+  "page.landing.description": "Apprenez Regex de manière interactive, entraînez-vous à votre niveau, testez et partagez vos propres Regex.",
+  "page.learn.description": "Apprenez Regex étape par étape. Suivez le tutoriel interactif qui rendra votre apprentissage plus facile et plus rapide.",
+  "page.cheatsheet.description": "Parfois, vous avez besoin de vous souvenir de Regex. Essayez l'antisèche qui vous permettra de mieux comprendre et de mieux mémoriser.",
+  "page.playground.description": "Vous pouvez facilement construire et tester votre motif regex en ligne via Playground.",
+
+  "learn.safari.unsupportWarning": "Les expressions régulières de cette étape ne sont pas prises en charge par le navigateur Safari. Si vous avez des difficultés à passer cette étape. Vous pouvez cliquer ici."
```

Comment:
The original of this text has been changed. The translation needs updating.

`Regex in this step are not supported by the some browsers. If you have trouble getting past this step. You can click here.`